src/extension.ts

```diff
@@ -94,6 +104,29 @@ export async function activateInternal(ctx: vscode.ExtensionContext, perfStats:
             ext.orchestratorManager = new OrchestratorRuntimeManager()
         );
 
+        if (ext.diagnosticLogging) {
+            try {
+                ext.outputChannel.appendLine(`DOCKER_HOST: ${process.env.DOCKER_HOST}`);
+            } catch {
+                // Do not throw for diagnostic logging
+            }
+        }
+
+        // Additional diagnostic logging for Linux systems
+        if (ext.diagnosticLogging && isLinux()) {
+            try {
+                await execAsync('uname -a');
```

Comment:
Consider adding a small utility function that would let you write this code in one line, e.g.

```ts
noThrow(() => await execAsync('uname -a"));
```

Reply:
Realized trying to implement this pattern as a method would be pretty tricky if not impossible due to typescript/eslint requirements for promise handling, so I'll leave as explicit try/catch blocks.